### ISSUE TITLE
Cartpole Policy Gradient script does not converge (2-cartpole/3-reinforce/cartpole_reinforce.py)

### ISSUE THREAD

**Issue by a-ozbek** [rlcode/reinforcement-learning#108]
I am running the script [here](https://github.com/rlcode/reinforcement-learning/blob/master/2-cartpole/3-reinforce/cartpole_reinforce.py) but even after 500 episodes it does not converge. You can see the graph I get below:
![score](https://user-images.githubusercontent.com/14084682/129976668-ec03b85d-e44f-429c-a59a-7a7354be4b72.png)

In contrast this is the supposedly converged graph from repo:
![cartpole_reinforce](https://user-images.githubusercontent.com/14084682/129976855-def0753e-0400-4d52-a0f3-5d6fe2f9bdd3.png)

Can you please advise why this is the case? I did not change any parameters, just ran the script as it is.

**Comment by dnddnjs** [comment]
Thanks for the report. The file/algorithm this issue references was removed or rewritten during the #122 modernization (TF/Keras → PyTorch, curated algorithm set of 9). The original code path no longer exists, so closing as no longer applicable. Feel free to reopen if you encounter a similar issue on the current PyTorch implementation.
